src/transformers/integrations.py

```diff
@@ -932,41 +932,38 @@ def __del__(self):
-        log_parameters (`bool`, *optional*, defaults to True):
-            If True, log all trainer arguments and model parameters provided by the trainer.
-        log_checkpoints (`str`, *optional*, defaults to "best"):
+        api_token (`str`, optional):
+            Neptune API token obtained upon registration. You can leave this argument out if you have
+            saved your token to the `NEPTUNE_API_TOKEN` environment variable (strongly recommended).
+            See full setup instructions in the [docs](https://docs.neptune.ai/getting-started/installation).
+        project (`str`, optional):
+            Name of an existing Neptune project, in the form: "workspace-name/project-name".
+            You can find and copy the name from the project Settings -> Properties in Neptune.
+            If None (default), the value of the `NEPTUNE_PROJECT` environment variable will be used.
+        name (`str`, optional):
+            Custom name for the run. It is shown in the Neptune app and stored in the "sys/name" field.
+        base_namespace (`str`, optional, defaults to "finetuning"):
+            Root namespace within the Neptune run.
+        log_parameters (`bool`, optional, defaults to True):
+            If True, logs all trainer arguments and model parameters provided by the trainer.
+        log_checkpoints (`str`, optional, defaults to "best"):
             If "same", upload checkpoints whenever they are saved by the trainer.
             If "best", upload the best checkpoint (among the ones saved by the trainer,
-            chosen at the end of the training)
+                chosen at the end of the training).
             Otherwise, do not upload checkpoints.
-        run (`Run`, *optional*):
-            Pass Neptune run object if you want to continue logging to the existing run (e.g., resume a run).
-            Read more about it `here <https://docs.neptune.ai/how-to-guides/neptune-api/resume-run>`_.
-            When run object is passed you can't specify other neptune properties.
-        **neptune_run_kwargs (*optional*):
+        run (`Run`, optional):
+            Pass a Neptune run object if you want to continue logging to an existing run.
+            Read more about resuming runs in the [docs](https://docs.neptune.ai/how-to-guides/neptune-api/resume-run).
+            When run object is passed, you can't specify other Neptune properties.
+        **neptune_run_kwargs (optional):
             Additional keyword arguments to be passed directly
-            to the `neptune.init() <https://docs.neptune.ai/api-reference/neptune#init>`_ function
+            to the [neptune.init_run()](https://docs.neptune.ai/api-reference/neptune#.init_run) function
             when a new run is created.
-            Note that `**neptune_run_kwargs` are passed down to `neptune.init()` only if run is not provided.
+            Note that `**neptune_run_kwargs` are passed to `neptune.init()` only if a `run` argument is not provided.
```

Comment:
That's isn't true (one comment above) -> it will be applied to every newly created Run (`run` could be still provided).